src/Shared/FileUtilities.cs

```diff
@@ -1179,5 +1179,13 @@ private static bool PathsEqualNonAscii(string strA, string strB, int i, int leng
 
             return false;
         }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        internal static void ClearFileExistenceCache()
+        {
+            FileExistenceCache.Clear();
```

Comment:
FileExistenceCache is not there in MSBuildTaskHost. IfDef the clear method in `CLR2COMPATIBILITY`